app/models/like.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: likes
+#
+#  id         :bigint           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  fan_id     :bigint           not null
+#  photo_id   :bigint           not null
+#
+# Indexes
+#
+#  index_likes_on_fan_id    (fan_id)
+#  index_likes_on_photo_id  (photo_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
+
+
+  validates :user_id, uniqueness: { scope: :photo_id, message: "has already liked this photo" }
```

Comment:
@APatel-AI you have a validation on a field named `user_id`, but in your like model you do not have this field. Can you update this?